### PR TITLE
Remove BLAKE-512 from auth header description

### PR DESCRIPTION
**Pull request by BLR-0118**
#196

### CODE REVIEW

**Comment by core-wg-admin** [comment]
Can we add some content on how to generate the `signing_string`. Maybe a link to an RFC for Signing and Verification?

**Comment by BLR-0118** [comment]
> Can we add some content on how to generate the `signing_string`. Maybe a link to an RFC for Signing and Verification?

Is such an RFC available or can I link this [doc](https://docs.google.com/document/d/1Iw_x-6mtfoMh0KJwL4sqQYM0kD17MLxiMCUOZDBerBo/edit#heading=h.zs1tt1ewtdt)?

**Comment by BLR-0118** [comment]
If ok, I can have a markdown version of the doc created for signing and add to the RFCs?

**Comment by core-wg-admin** [comment]
@BLR-0118 That would be great

**Comment by BLR-0118** [comment]
Added RFC for signing & verification - docs/protocol-drafts/BECKN-RFC-006-Signing-Beckn-APIs-In-HTTP-Draft-01.md

**Review by core-wg-admin — APPROVED** [review]

File: docs/protocol-drafts/BECKN-RFC-006-Signing-Beckn-APIs-In-HTTP-Draft-01.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+**Headers:**
+
+HTTP/1.1 401 Unauthorized
+
+Proxy-Authenticate: Signature realm="example-bpp.com",headers="(created) (expires) digest"
+
+...
+
+**Request Body:**
+
```
Comment (core-wg-admin) [inline docs/protocol-drafts/BECKN-RFC-006-Signing-Beckn-APIs-In-HTTP-Draft-01.md:200]: Can you please fix the code blocks using the "```" tags like this?

```
{
 "message": {
    "ack": {
        "status": "NACK"
    }
  }
}
```

Comment (core-wg-admin) [reply docs/protocol-drafts/BECKN-RFC-006-Signing-Beckn-APIs-In-HTTP-Draft-01.md:200]: @BLR-0118

**Review by core-wg-admin — APPROVED** [review]

**Pull request merged**